config_example.yaml

```diff
@@ -15,6 +15,7 @@ slack:
   outputformat: "all" # all (default), text, fields
   minimumpriority: "" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
   #messageformat: 'Alert : rule *{{ .Rule }}* triggered by user *{{ index .OutputFields "user.name" }}*' # a Go template to format Slack Text above Attachment, displayed in addition to the output from `SLACK_OUTPUTFORMAT`, see [Slack Message Formatting](#slack-message-formatting) in the README for details. If empty, no Text is displayed before Attachment.
+  checkcert: true # check if ssl certificate of the output is valid (default: true)
```

Comment:
```suggestion
```

`Slack` only provides httpS with trusted certificates, we should not disable the check. (You didn't for Discord for example).